test-context/src/lib.rs

```diff
@@ -126,11 +126,25 @@ impl AsyncTestContext for TrustifyContext {
             return TrustifyContext::new(db, None).await;
         }
 
-        let (db, postgresql) = db::embedded::create()
-            .await
-            .expect("Create an embedded database");
-
-        TrustifyContext::new(db, postgresql).await
+        if env::var("DB_PATH").is_ok() {
+            let db_path: String = env::var("DB_PATH")
+                .unwrap_or(".".into())
+                .parse()
+                .expect("a text value");
+            let path = Path::new(&db_path);
+
+            let (db, postgresql) = db::embedded::create_in(&path, false)
+                .await
+                .expect("Create an embedded database");
+
+            TrustifyContext::new(db, postgresql).await
+        } else {
+            let (db, postgresql) = db::embedded::create(true)
+                .await
+                .expect("Create an embedded database");
+
+            TrustifyContext::new(db, postgresql).await
+        }
```

Comment:
Again, I don't think this code justifies its existence.

Reply:
> Again, I don't think this code justifies its existence.

ok I'm +1 here...

a) because of upload issue I would like to see the DB logs after run `cargo test`
b) I sent this PR because of upload issue...
c) I saw the logs and it was exactly the same (the error message) as the app

I'll close the PR :+1: